File: Assignments/Brea/Class Examples/Test_051520.js
//Test Examples in Python from May 15th, 2020
//JavaScript version of Pick 6 Lab

function randInt(lowerNum, higherNum) {
    let leftOffset = lowerNum
    let middleArea = (higherNum - lowerNum) + 1 // like saying randint(2, 5)
    return leftOffset + middleArea * Math.floor(Math.random())
}

function getTicket() {
    let output = []
    for (let i=0; i<6, i++){
        output.push(randInt(1, 5))
    }
    return output
}

File: Assignments/Brea/Class Examples/Test_051520.py
#Test Examples May 15th, 2020

def get_ticket():
    output = []
    for in in range(6):
        output.append(random.randint(1, 99))
    return [random.randint(1, 99) for i in range(6)]

winning_ticket = get_ticket()
for i in range(100):
    matches = 0
    user_ticket = get_ticket()
    for i in range(len(winning_ticket)):
        if winning_ticket[i] == user_ticket[i]:
            matches += 1
        print(matches)
